pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -201,7 +201,7 @@ public NamespaceService(PulsarService pulsar) {
         this.redirectManager = new RedirectManager(pulsar);
 
         this.lookupLatencyHistogram = pulsar.getOpenTelemetry().getMeter()
-                .histogramBuilder("pulsar.broker.lookup.request.duration")
+                .histogramBuilder(LOOKUP_REQUEST_DURATION_METRIC_NAME)
```

Comment:
The description should match how OTel writes it: "The duration of topic lookup requests (either binary or HTTP)". @codelipenghui Am I right?

Reply:
Done!